### ISSUE TITLE
Python 3.3 segfaults when trying to import

### ISSUE THREAD

**Issue by zetok** [aitjcize/PyTox#9]
```
print("So far it works")
from tox import Tox, ToxAV
print("Succesfully imported Tox")
```
results in:
```
$ ./botze.py 
So far it works
Segmentation fault
```

**Comment by kubov** [comment]
I have the same problem (gentoo linux, 64bit). Surprisingly, bug is not present while using virtualenv environment.
stack trace:
```
#0  0x00007ffff74d6a5a in strlen () from /lib64/libc.so.6
#1  0x00007ffff7aa5b30 in PyString_FromString () from /usr/lib64/libpython2.7.so.1.0
#2  0x00007ffff7a90781 in PyDict_GetItemString () from /usr/lib64/libpython2.7.so.1.0
#3  0x00007ffff7ab19be in PyType_Ready () from /usr/lib64/libpython2.7.so.1.0
#4  0x00007ffff0209e28 in initpytox () from /usr/lib64/python2.7/site-packages/pytox.so
#5  0x00007ffff7b03247 in _PyImport_LoadDynamicModule () from /usr/lib64/libpython2.7.so.1.0
#6  0x00007ffff7b012e1 in ?? () from /usr/lib64/libpython2.7.so.1.0
#7  0x00007ffff7b0155f in ?? () from /usr/lib64/libpython2.7.so.1.0
#8  0x00007ffff7b01f3e in PyImport_ImportModuleLevel () from /usr/lib64/libpython2.7.so.1.0
#9  0x00007ffff7ae7988 in ?? () from /usr/lib64/libpython2.7.so.1.0

```

**Comment by alexandervdm** [comment]
Re-compiled net-im/tox and dev-python/pytox on my unhardened Gentoo AMD64 machine with Python 3.3.6 and no problems for me. 

I did have to change the original test program in the opening post to match the new module name 'pytox'. You guys aren't importing some old version of the 'tox' module that got left behind in your installation right?

print("So far it works")
from **pytox** import Tox, ToxAV
print("Succesfully imported Tox")

**Comment by kitech** [comment]
see the related fix PR #51 , this should be also fixed .

**Comment by kitech** [comment]
see the related fix PR #51 , this should be also fixed .
